diff -up kontact-20.03.90/src/kcmkontact.cpp.omv~ kontact-20.03.90/src/kcmkontact.cpp
--- kontact-20.03.90/src/kcmkontact.cpp.omv~	2020-04-13 01:46:06.376802107 +0200
+++ kontact-20.03.90/src/kcmkontact.cpp	2020-04-13 01:46:10.656849943 +0200
@@ -106,11 +106,7 @@ PluginSelection::PluginSelection(KConfig
               "Select the plugin from this drop down list to be used as the "
               "initial plugin each time Kontact is started. Otherwise, Kontact "
               "will restore the last active plugin from the previous usage."));
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
     connect(mPluginCombo, qOverload<int>(&QComboBox::currentIndexChanged), this, &PluginSelection::changed);
-#else
-    connect(mPluginCombo, qOverload<int, const QString &>(&QComboBox::currentIndexChanged), this, &PluginSelection::changed);
-#endif
 }
 
 PluginSelection::~PluginSelection()
